scripts/init_chain.sh

```diff
@@ -11,7 +11,8 @@ if ! command_exists jq; then
 fi
 
 rm -rf ~/.lava
-lavad init validator --chain-id lava
+chainID="lava-local-1"
```

Comment:
why? what was wrong with the original chain-id "lava"?

Reply:
omer want to differentiate it from mainnet chain id so there wont be mix ups